core/src/main/java/org/apache/calcite/sql/util/SqlShuttle.java

```diff
@@ -57,6 +58,11 @@
     return param;
   }
 
+  // TODO: FIXME?
```

Comment:
what's the issue here?

Reply:
I don't think there is an issue. I was marking this as a possible place to look if I encountered issue.